src/consts.hpp

```diff
@@ -8,9 +8,9 @@ namespace gpudrive {
 
 namespace consts {
 
-inline constexpr madrona::CountT kMaxAgentCount = 128;
-inline constexpr madrona::CountT kMaxRoadEntityCount = 6000;
-inline constexpr madrona::CountT kMaxAgentMapObservationsCount = 200;
+inline constexpr madrona::CountT maxNumObjects = 128;
+inline constexpr madrona::CountT maxNumRgPoints = 6000;
+inline constexpr madrona::CountT roadGraphTopK = 200;
```

Comment:
I think the original names are more meaningful in the simulator code base.

- The "k" prefix is a common naming convention for `const` variables. See, for instance, Google's [style guide](https://google.github.io/styleguide/cppguide.html#Constant_Names). 
- `maxNumObjects` doesn't distinguish between agents and road points, but it can only refer to the agent count
- Each road entity is actually a line segment, not a point as `maxNumRgPoints` suggests.
- We don't use the term "roadGraph" (or "Rg") anywhere in simulator IIRC, so introducing it here may be confusing

You can use pybind to export these constants under new names if they are more meaningful in learning context.